raster-foundry/raster-foundry
# Remove All References to Airflow

## Overview

Removes all references to Airflow in builds, etc.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Notes

We made a bad mistake of using a "machine" user called `rf|airflow-user` in many positions. This is going to take a little extra care to excise.

Also - the container for the batch jar is still rather large -- this isn't ideal, but also seems a little bit out of scope for removing Airflow and a little risky to make additional changes with what is already a pretty big change.

## Testing Instructions

 * Start server, interact with UI, etc and ensure everything functions as normal

Closes #2507
